platforms/android/tangram/src/main/cpp/androidPlatform.cpp

```diff
@@ -63,8 +63,15 @@ void AndroidPlatform::bindJniEnvToThread(JNIEnv* jniEnv) {
     jniEnv->GetJavaVM(&jvm);
 }
 
-void AndroidPlatform::setupJniEnv(JNIEnv* jniEnv) {
-    bindJniEnvToThread(jniEnv);
+jint AndroidPlatform::jniOnLoad(JavaVM* javaVM) {
```

Comment:
Or does these get GC'ed automatically? Still it would be prudent to delete these.

Reply:
Updated this to delete the global ref on the hasmap class in OnUnload. The tangramClass ref is handled by the dispose call and would be difficult to delete from a free function like this because it's a member of the AndroidPlatform class.